trimmerdecorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? "#{@nameable.correct_name[0..10]}..." : @nameable.correct_name
```

Comment:
- [ ] It will be great to trim the correct name to have a maximum of 10 characters. In your implementation, you have trimmed from [0..10]. This is wrong because it means your correct name will have 11 characters, not 10. Instead it should be `[0..9]`